openstack/sap-seeds/templates/flavor-seed.yaml

```diff
@@ -6,484 +6,400 @@ spec:
   requires:
   - monsoon3/nova-flavor-seed
   flavors:
+  - name: "forced"
+    id: "1"
+    vcpus: 1
+    ram: 1024
+    disk: 0
+    is_public: false
+    extra_specs:
+      {{- tuple . "forced" | include "sap_seeds.helpers.extra_specs" | indent 6 }}
```

Comment:
This renders as `extra_specs:` - are you sure this will work with the seeder? Shouldn't it maybe be `extra_specs: {}` or something to have the right type?

Reply:
It looks it doesn't matter. The parsed results are looking like this:
```
{'name': 'forced', 'id': '1', 'ram': 1024, 'vcpus': 1, 'is_public': False}
```
It may be due to the `omitempty` annotation for the [field](https://github.com/sapcc/openstack-seeder/blob/2b1a09bd9676575277640fecdfb512f874e66687/api/v2/openstackseed_spec.go#L229).